CMakeLists.txt

```diff
@@ -484,6 +484,22 @@ if(HAVE_ATTRIBUTE_ALIGNED)
     add_definitions(-DHAVE_ATTRIBUTE_ALIGNED)
 endif()
 
+#
+# Check for __builtin_assume_aligned(x,n) support in the compiler
+#
+check_c_source_compiles(
+    "char *test(char *buffer) {
+         char *abuffer = __builtin_assume_aligned(buffer,64);
```

Comment:
Inconsistent padding.

Reply:
Is it though? I don't see it.

Reply:
I see 9 spaces before `char *abuffer` and 8 before `return`.

Reply:
There it was, thanks 🤦‍♂️